docs/UserGuide.md

```diff
@@ -247,11 +242,11 @@ the rooms that have room clean statuses set to "no".
 #### Format: `find KEYWORD [MORE_KEYWORDS]`
 
 #### Constraints of parameters:
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only full words will be matched e.g. `Han` will not match `Hans`
+* The search is case-insensitive. E.g., `hans` will match `Hans`
+* The order of the keywords does not matter. E.g., `Hans Bo` will match `Bo Hans`
+* Only full words will be matched E.g., `Han` will not match `Hans`
 * Guests matching at least one keyword will be returned (i.e. `OR` search).
```

Comment:
Should this be I.e.?

Reply:
Similarly, not at the start of a sentence so lowercase is okay